### DISCUSSION TITLE
Change Icon when in TODO mode

### DISCUSSION THREAD

**Discussion by LeonardoGentile** [rxhanson/Rectangle#1124] [Q&A]
Hello, thanks for your work, extremely useful app 👍
I really like the todo mode, but I always forget if I am in todo mode or not once I start to resize windows here and there.

I was wondering if there is a settings or maybe introduce a new feature to give us a hint we are in todo mode or not.
I was thinking for example to change the menu bar icon color when todo mode is active.

What do you think?

**Comment by rxhanson** [comment]
I don't have the bandwidth to add this in at the moment, but see no reason to turn down a good pull request for this.
